react-frontend/src/App.test.js

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import App from './App';
+
+test('renders learn react link', () => {
+  const { getByText } = render(<App />);
+  const linkElement = getByText(/learn react/i);
```

Comment:
@jainpawan21 This test will fail as `<App />` doesn't have `learn react` text anymore. Please update this.

Reply:
Removed App.test.js!